src/vcpkg/configure-environment.cpp

```diff
@@ -138,13 +128,15 @@ namespace vcpkg
         msg::println(Color::warning,
                      LocalizedString::from_raw("Using in-development vcpkg-artifacts built at: ").append_raw(ce_path));
 #else  // ^^^ VCPKG_ARTIFACTS_PATH / give up and always download latest vvv
-        fs.remove(ce_sha_path, VCPKG_LINE_INFO);
-        fs.remove_all(ce_path, VCPKG_LINE_INFO);
+        auto base_path = paths.root / "artifacts";
```

Comment:
does this still need to be under VCPKG_ROOT or should it also be found under the cache?

Reply:
Fixed!